src/main/resources/network/brightspots/rcv/hints_winning_rules.txt

```diff
@@ -34,6 +34,8 @@ Tiebreak Mode (required): Ties in ranked-choice voting contests can occur when e
 
   * Previous round counts (then random): The tied candidate with the least votes in the previous round loses the tie. If there is a tie in the previous round, the tie is broken randomly. Requires a random seed.
```

Comment:
Suggest:
> The tied candidate with the least votes in the previous round loses the tie. If there is a tie in the previous round, it looks at the round before that. If there are still tied candidates after examining all previous rounds, the tie is broken randomly. Requires a random seed.

This and following suggestions repeat text. Could restructure these to say instead:
> Looks at previous rounds to break ties (see below). If there are still tied candidates after examining previous rounds, the tie is broken randomly. Requires a random seed.